src/main/java/com/mattmalec/pterodactyl4j/client/entities/ClientSubuser.java

```diff
@@ -22,11 +22,31 @@
 import java.time.OffsetDateTime;
 import java.util.UUID;
 
+/**
+ * Represents a sub-user pm a {@link ClientServer}
+ */
 public interface ClientSubuser extends User, IPermissionHolder {
-
+    /**
+     * Gets the avatar link of the {@link ClientSubuser}
+     *
+     * @return link to image of the avatar
+     */
     String getImage();
+
+    /**
+     * Checks if the {@link ClientSubuser} has 2fa enabled
+     *
+     * @return whether the user has 2fa enabled
+     */
     boolean has2FA();
+
+    /**
+     * Gets the uuid of the {@link ClientSubuser}
```

Comment:
The description and return value are kinda recursive here and explain the same thing. It's important to notate if this method can return null at all.
```suggestion
	 * The UUID of the Server.
	 *
	 * @return Never-null {@link java.util.UUID} containing the Subuser's UUID.
	 */
	UUID getUUID();
```